### PR TITLE
README.md: add update instructions

### PR DESCRIPTION
**Pull request by Ivansete-status**

### CODE REVIEW

**Review by NagyZoltanPeter — APPROVED** [review]
Great notice

**Review by gabrielmer — APPROVED** [review]
Thanks so much!

File: README.md [hunk 1]
```diff
@@ -74,3 +74,13 @@ curl -X GET "http://127.0.0.1:8645/store/v1/messages?contentTopics=%2Fmy-app%2F2
 ```
 
 For advanced documentation, refer to [ADVANCED.md](https://github.com/waku-org/nwaku-compose/blob/master/ADVANCED.md).
+
+-----
+# How to update to latest version
+
+We regularly announce new available versions in our [Discord](https://discord.com/channels/1110799176264056863/1111568123007606855) server.
```
Comment (fryorcraken) [inline README.md:81]: I would add a link to invite: https://discord.waku.org/

**Comment by fryorcraken** [comment]
Yet, this does not explain how to migrate to the latest with RLNv2...

**Pull request merged**